pom.xml

```diff
@@ -60,6 +60,9 @@
         <jenkins.version>2.60.3</jenkins.version>
         <java.level>8</java.level>
         <workflow.version>2.0</workflow.version>
+        <node.version>10.17.0</node.version>
```

Comment:
is there a reason it's using old node? new one doesn't work with the tests?

Reply:
This is the maximum version that works with gulp

Reply:
😂